### DISCUSSION TITLE
Release versions also patches

### DISCUSSION THREAD

**Discussion by cesxhin** [Baroshem/nuxt-security#442] [Ideas]
I would like it to track all release versions even those patches.

**Comment by Baroshem** [comment]
Hey, thanks for starting this discussion.

We intentionally don't do releases for patches because we usually release patches to fix certain issue immediately (i.e. after recent release we figured out that there was a bug that was only discoverable after the package was published).

For such cases, there is no need to publish a release note. We do it however for all minor and major releases where there is a lot of bugfixes, new features, docs, and so on.
